generateInitialKey/utils/generateAddress.js

```diff
@@ -0,0 +1,19 @@
+const keythereum = require("keythereum");
+
+//generates initial key keystore file
+function generateAddress(password) {
+	return new Promise((resolve, reject) => {
+		let params = { keyBytes: 32, ivBytes: 16 };
+
+	  	let dk = keythereum.create(params);
+
+	  	keythereum.create(params, function (dk) {
```

Comment:
`dk` here shadows `dk` defined above (at line 8). Either this one should be renamed, or the one above is never used.